torchgeo/datasets/cdl.py

```diff
@@ -135,3 +137,50 @@ def _extract(self) -> None:
         pathname = os.path.join(self.root, self.zipfile_glob)
         for zipfile in glob.iglob(pathname):
             extract_archive(zipfile)
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`RasterDataset.__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        .. versionadded:: 0.3
```

Comment:
Technically, this isn't true. This method also existed in 0.1. It might be more correct to use `versionchanged` with a description of what changed (method now takes a sample dict, not a Tensor, additional kwargs added, etc.). This will need to be changed in all the "add plot method" PRs.